README.md

```diff
@@ -627,6 +627,62 @@ In addition to the options provided in the base `OpenAI` client, the following o
 
 An example of using the client with Microsoft Entra ID (formerly known as Azure Active Directory) can be found [here](https://github.com/openai/openai-python/blob/main/examples/azure_ad.py).
 
+## Activating CLI Autocomplete
+Use the following commands to activate CLI autocompletion for each shell.
+
+Install `openai` with `[cli]` option to install argcomplete.
+```shell
+pip install openai[cli]
+```
+
+### Bash & Zsh
```

Comment:
Are there existing docs we could link to here instead of listing out all the possible environments here? maybe argcomplete's docs?

Reply:
Both temporary and permanent argcomplete activation instructions for Fish and Powershell are available on argcomplete's [contrib directory](https://github.com/kislyuk/argcomplete/tree/develop/contrib). However, only the temporary activation commands for Bash and Zsh are listed on argcomplete's [README file.](https://github.com/kislyuk/argcomplete/tree/develop?tab=readme-ov-file#synopsis) The commands to activate autocompletion permanently for Bash and Zsh are not found anywhere on argcomplete's docs.

What are your opinions on replacing all the other environment activation instructions with the appropriate links to argcomplete's docs, while keeping these lines for permanent activation on Bash and Zsh?

For Bash:
```shell 
register-python-argcomplete openai >> ~/.bashrc
```
For Zsh:
```shell  
register-python-argcomplete openai >> ~/.zshrc
```

Reply:
cc @kwhinnery-openai for docs review

Reply:
Hello @RobertCraigie, we have refactored our section of the readME file to utilize existing documentation from argcomplete. Let us know if anything else needs to be fixed.